static/js/settings_org.js

```diff
@@ -15,6 +15,10 @@ exports.reset = function () {
 };
 
 exports.maybe_disable_widgets = function () {
+    if (page_params.realm_plan_type !== 4) {
```

Comment:
Here too, we can avoid hardcoding values, and put a constant in `settings_config` if there isn't one.